crates/bevy_ecs/src/world/entity_ref.rs

```diff
@@ -534,6 +495,49 @@ impl<'w> EntityMut<'w> {
     }
 }
 
+impl<'w> EntityMut<'w> {
+    /// **You should prefer to use the typed API [`EntityMut::get`] where possible and only
```

Comment:
Because of how IDE support for doc strings works, this warning should be be in the second paragraph to ensure that the tooltips remain useful.